CODEOWNERS

```diff
@@ -1,8 +1,12 @@
-*				@labkode @ishank011
-pkg/auth/manager/owncloudsql	  @cs3org/owncloud-team
-pkg/storage/fs/owncloudsql      @cs3org/owncloud-team
-pkg/storage/fs/owncloud   	  @cs3org/owncloud-team
-pkg/storage/fs/ocis             @cs3org/owncloud-team
-pkg/storage/utils/decomposedfs  @cs3org/owncloud-team
-pkg/user/manager/owncloudsql	  @cs3org/owncloud-team
-pkg/app           		  @wkloucek @glpatcern @ishank011
+*                                     @labkode @ishank011
+changelog                             @glpatcern @cs3org/owncloud-team
+internal/http/services/appprovider    @wkloucek @glpatcern
```

Comment:
@glpatcern for these rules I think you need to add @labkode and me separately, because it ignores hierarchical ownership